SETUP/ARCHIVING.md

```diff
@@ -0,0 +1,51 @@
+# Project Archiving
+
+After projects are completed, they can be archived. This includes moving the
+project files into a separate archive directory, moving the project tables into
+a separate archive database, and moving a project's page_events and
+wordcheck_events into the archive database.
+
+The database instructions in this file resemble those for the main database. See
+[INSTALL.md](INSTALL.md) for more information on how to run database commands.
+
+To enable archiving you must set up the archive database and the archive
+project directory.
+
+1. Set the two archive settings in `configuration.sh`:
+
+   * `_ARCHIVE_DB_NAME` sets the name of the archive database. In the rest of
+     this document we use the name `dp_archive` for this value.
+
+   * `_ARCHIVE_PROJECTS_DIR` sets the directory where project filesystem
+     artifacts are moved.
+
+2. Run the `configure` script to update the site configuration with the new
+   archive settings. See [INSTALL.md](INSTALL.md) for more information on how
+   to do this.
+
+3. Create the database and grant it access using the **same credentials used for
+   the primary database**. From within a `mysql` client session:
+   ```sql
+   CREATE DATABASE dp_archive CHARACTER SET utf8mb4;
+   GRANT ALL  ON dp_archive.* TO dp_user@localhost IDENTIFIED BY 'dp_password';
+   ```
+
+4. Create the `page_events` and `wordcheck_events` table in the archive
+   database. Find the schema for those tables in `db_schema.sql` and use them
```

Comment:
Should we create a separate schema for these shared tables and have the db_schema.swl import it?

Reply:
Unfortunately we have several tools that use the `db_schema.sql` directly, so this wouldn't work.